src/components/AnalysisPage/ThicknessAnalysisBox/StartButton/StartButton.jsx

```diff
@@ -1,11 +1,19 @@
 import { Button } from "antd";
 import "./StartButton.css";
 import PropTypes from "prop-types";
+import { startThicknessAnalysis } from "./startThickAnalysis";
 
-const StartButton = ({ partId }) => {
+const StartButton = ({ partId, thresholdValue }) => {
   const StartAnalysis = async () => {
-    console.log("Starting analysis");
-    console.log(partId);
+    console.log("Starting thickness analysis...");
+    try {
+      const job = await startThicknessAnalysis(partId, thresholdValue);
+
+      console.log(job);
+    } catch (error) {
+      console.error("Failed to start thickness analysis.", error);
+      return;
```

Comment:
This log is basically just going print the same message twice because your startThicknessAnalysis function throws the message 'Failed to start thickness analysis' and then you log that same message here.

Reply:
all good here now!